iscsi/iscsi.go

```diff
@@ -397,33 +415,84 @@ func DisconnectVolume(c Connector) error {
 	return nil
 }
 
-// RemovePhysicalDevice removes device(s) sdx from a Linux host.
-func RemovePhysicalDevice(devices ...string) error {
-	debug.Printf("Removing scsi device %v.\n", devices)
+func GetScsiDevice(device string) (*ScsiDevice, error) {
+	scsiDevices, err := GetScsiDevices(device)
+	if err != nil {
+		return nil, err
+	}
+	return &scsiDevices[0], nil
+}
+
+// GetScsiDevices get scsi devices from a device names
+func GetScsiDevices(devices ...string) ([]ScsiDevice, error) {
```

Comment:
we can actually have one function in place of `GetSCSIDevice or GetSCSIDevices()`and keep the logic working for both ( one device or devices)

Reply:
I didn't find any instance if `GetSCSIDevice`, I guess it has already been updated.

Reply:
yeah, may be I looked at cached version at my end.